### PR TITLE
Fix release-deb script

### PR DESCRIPTION
**Pull request by mlaventure**
ping @tianon @tiborvass 

Fixes https://github.com/docker/docker/issues/23180 (well, the files would have to be regenerated)

### CODE REVIEW

File: hack/make/release-deb [hunk 1]
```diff
@@ -29,7 +29,8 @@ arches=( amd64 i386 )
 
 # Preserve existing components but don't add any non-existing ones
 for component in main testing experimental ; do
-	if ls "$APTDIR/dists/*/$component" >/dev/null 2>&1 ; then
+	exists=$(find "$APTDIR/dists" -mindepth 2 -maxdepth 2 -type d -name $component -print -quit)
```
Comment (tianon) [inline hack/make/release-deb]: let's throw some quotes around `$component` here, but otherwise :+1:

Comment (tianon) [inline hack/make/release-deb]: If you want, we could also skip the var entirely, ie:
```bash
if [ -n "$(find "$APTDIR/dists" -mindepth 2 -maxdepth 2 -type d -name "$component" -print -quit)") ]; then
```

Comment (mlaventure) [inline hack/make/release-deb]: yeah, just found it easier to read with the var, but either way works for me

**Comment by mlaventure** [comment]
@tianon updated

**Comment by tiborvass** [comment]
LGTM

**Comment by tianon** [comment]
LGTM :+1:

**Comment by thaJeztah** [comment]
looks like enough LGTM's, merging \o/

**Pull request merged**